lib/Collection/QueryType/Handler/TaxonHandler.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Layouts\Sylius\Collection\QueryType\Handler;
+
+use Netgen\Layouts\API\Values\Collection\Query;
+use Netgen\Layouts\Collection\QueryType\QueryTypeHandlerInterface;
+use Netgen\Layouts\Parameters\ParameterBuilderInterface;
+use Netgen\Layouts\Parameters\ParameterType;
+use Netgen\Layouts\Sylius\Parameters\ParameterType as SyliusParameterType;
+use Sylius\Component\Taxonomy\Model\TaxonInterface;
+use Sylius\Component\Taxonomy\Repository\TaxonRepositoryInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+use function count;
+use function trim;
+
+class TaxonHandler implements QueryTypeHandlerInterface
+{
+    /**
+     * @param TaxonRepositoryInterface<TaxonInterface> $taxonRepository
+     */
+    public function __construct(
+        private readonly TaxonRepositoryInterface $taxonRepository,
```

Comment:
Do not use `private readonly`, we're not using this construct here. `private TaxonRepositoryInterface $taxonRepository` is enough.